django-stubs/contrib/auth/views.pyi

```diff
@@ -7,7 +7,7 @@ from django.http.response import HttpResponse, HttpResponseRedirect
 from django.views.generic.base import TemplateView
 from django.views.generic.edit import FormView
 
-UserModel = get_user_model()
```

Comment:
Same as the comment above, this exists at runtime and causes issues with stubtest